### PR TITLE
wasmedge 0.14.0

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/w/wasmedge.rb [hunk 1]
```diff
@@ -17,16 +17,12 @@ class Wasmedge < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "ninja" => :build
```
Comment (chenrui333) [inline Formula/w/wasmedge.rb:20]: why is ninja removed?

Comment (cho-m) [reply Formula/w/wasmedge.rb:20]: Our preference in Homebrew/core has been Unix Makefiles for CMake unless there is a strong reason for Ninja (e.g. Unix Makefile generation is broken for specific repo).

Comment (chenrui333) [reply Formula/w/wasmedge.rb:20]: aha, I was not aware that makefile over ninja preference, any docs links?

Comment (cho-m) [reply Formula/w/wasmedge.rb:20]: I don't think it was explicitly documented but I recall some comments/discussions before on this.

Our default choice in `brew create` can be considered the guideline, i.e. `cmake` will use `make` (official default) and `meson` will use `ninja` (official default). And perhaps the majority of existing formulae.

A large portion of Ninja performance is from incremental builds which isn't useful in Homebrew builds and we avoid the extra dependency.

Comment (chenrui333) [reply Formula/w/wasmedge.rb:20]: yeah, that actually makes sense to me 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**